NCS.DSS.Customer/SearchCustomerHttpTrigger/Service/SearchCustomerHttpTriggerService.cs

```diff
@@ -1,19 +1,17 @@
 ﻿using NCS.DSS.Customer.Cosmos.Provider;
-using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
+using Microsoft.Azure.Search;
 
 namespace NCS.DSS.Customer.SearchCustomerHttpTrigger.Service
 {
     public class SearchCustomerHttpTriggerService : ISearchCustomerHttpTriggerService
     {
-        public async Task<List<Models.Customer>> SearchCustomerAsync(string givenName = null, string familyName = null, string dateofBirth = null,
-            string uniqueLearnerNumber = null)
+        public async Task<List<Models.Customer>> SearchCustomerAsync(ISearchIndexClient indexClient, string searchText,
+            string filter = null, IList<string> order = null, IList<string> facets = null)
         {
             var documentDbProvider = new DocumentDBProvider();
-            var customer = await documentDbProvider.SearchAllCustomer(givenName, familyName, dateofBirth, uniqueLearnerNumber);
+            var customer = await documentDbProvider.SearchCustomer(indexClient, searchText, filter, order, facets);
```

Comment:
Should this be wrapped in a try catch or is there one higher up?

Reply:
Theres a try catch on the DB Provider side where it call Azure Search